deep-security-agent/tasks/check-in-with-manager.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018, Trend Micro
+#
+# License as per [repo](master/LICENSE)
+#
+# *********************************************************************
+# * Ask the Deep Security agent to check-in
+# *********************************************************************
+---
+# *********************************************************************
+# * Check if DSA installed properly
+# *********************************************************************
+- debug:
+    msg: "Check if DSA installed properly."
+
+- stat:
+    path: /opt/ds_agent/dsa_control
+  register: dsaControl
+
+- fail:
+    msg: 
+        - "Could not find command /opt/ds_agent/dsa_control to operate DSA."
+        - "Please make sure Deep Security Agent has been installed"
+  when: dsaControl.stat.size is not defined 
+
+# *********************************************************************
+# * Ask the Deep Security agent to check in with the Deep Security manager
+# *********************************************************************
+- debug:
+    msg: "Ask the Deep Security agent to check in with the Deep Security manager" 
+
+- shell: /opt/ds_agent/dsa_control -m
+  when: ansible_os_family != "Windows"
+  become: true
```

Comment:
Why we need become:true here but no in the below condition? I think we can safely remove this part?

Reply:
"become" means you need to run that shell command with sudo. And in windows, we don't have to do this.